### PR TITLE
Notify on new review

### PR DESCRIPTION
**Pull request by DanielJackson-Oslo**
Adds a flash message when writing a new review, as well as a test for that flash message. 

Also fixes so the reviews section is reloaded with the new review even if there were no reviews there already, fixing the bug in https://trello.com/c/85WKlQl9/178-live-stream-av-nye-reviews-til-spaceshow-oppdaterer-ikke-n%C3%A5r-de-g%C3%A5r-fra-0-1

### CODE REVIEW

File: app/views/spaces/show/_reviews.html.erb [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-  <%= render partial: 'spaces/show/review_link_to_new_review', locals: {
-    space: @space
-  } %>
-</div>
-
+<div id="reviews">
+  <div class="flex gap-2 items-baseline justify-between">
+    <h2><%= Review.model_name.human(count: 2) %></h2>
+    <%= render partial: 'spaces/show/review_link_to_new_review', locals: {
+      space: @space
```
Comment (DanielJackson-Oslo) [inline app/views/spaces/show/_reviews.html.erb:5]: Trengs ikke sette local for @space , kan bare bruke den vi har

**Comment by DanielJackson-Oslo** [comment]
@mathias234 Ready for review

**Review by mathias234 — CHANGES_REQUESTED** [review]
I don't seem to be getting a flash message. But the reviews section does get the new entry

**Comment by DanielJackson-Oslo** [comment]
> I don't seem to be getting a flash message. But the reviews section does get the new entry

Hm... Will investigate. I even made a test for getting the flash message, so it might be faulty!

**Comment by DanielJackson-Oslo** [comment]
Fikset! Hadde klart å fjerne elementet som turbo stream sendte til når jeg merget inn main sine endringer.

@mathias234 Noen idé om hvordan teste at turbo streams fungerer som de skal? Trenger vi capybara-tester for det?

**Review by mathias234 — APPROVED** [review]
LGTM! 
Begge fungere nye reviews kommer opp i listen og jeg får en flash message 🚀

**Comment by mathias234** [comment]
> @mathias234 Noen idé om hvordan teste at turbo streams fungerer som de skal? Trenger vi capybara-tester for det?

Har funnet en guide på hvordan vi kan teste turbo stream med en request spec, men har aldri prøvd det selv https://github.com/mdominiak/hotwire-chat#testing

**Pull request merged**